src/Sulu/Bundle/ActivityBundle/Application/Subscriber/DispatchSpecificDomainEventSubscriber.php

```diff
@@ -41,6 +41,6 @@ public function dispatchDomainEventWithSpecificEventName(DomainEvent $event): vo
         // allows to register listeners that listen to all domain events.
         // this subscriber additionally dispatches the event with a specific event-name such as TagRemovedEvent::class
         // to allow for registering listeners for a specific type of event.
-        $this->eventDispatcher->dispatch($event, \get_class($event));
```

Comment:
get_class is the default behaviour of the event dispatcher that is why rector is replacing it.